test/markup/php/functions.txt

```diff
@@ -49,3 +49,15 @@ setAlarm(
     label: 'foo',
     time:time() + array(5)[0] + Foo::HOUR,
 );
+
+header(
```

Comment:
Is header a PHP thing?

Reply:
In this context (functions.txt), he calling user definer function:

> header()

Reply:
> Is header a PHP thing?

Yes! That is used to add a HTTP response header to the output. PHP is pretty much a language to generate web pages and their responses and `header()` is I guess known to anyone practicing PHP.  https://www.php.net/manual/en/function.header.php

The function name should not matter to highlight.js, as long as it is recognized as a function. If `header()` is confusing, I do not mind changing it to another built-in function that is common to other languages such as `printf()`.